railties/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Add benchmark generator
+
+    Introduce benchmark generator to benchmark Rails applications.
+
+      `rails generate benchmark opt_compare`
+
+    This creates a benchmark file that uses [`benchmark-ips`](https://github.com/evanphx/benchmark-ips).
+    By default, two code blocks can be benchmarked using the `before` and `after` reports.
+
+    You can run the generated benchmark file using:
+      `bundle exec ruby script/benchmarks/opt_compare.rb`
```

Comment:
We don't need `bundle exec`. `config/environment` setup bundler so we should remove this.